### DISCUSSION TITLE
Directory, File Permissions

### DISCUSSION THREAD

**Discussion by charlotattardjpj** [drakkan/sftpgo#536] [General]
Hello,

I have this setup on a CentOS machine. The below is an example of a directory and its files which is mounted on the SFTP. I have created a user on the SFTP with * Permission. When I login with this user I cannot move files, upload or delete them. Im using cyberduck as a client. It gives me an error that user does not have permissions.

Absolute Path : /sftp_drakkan/YY/NN/

-rwxrwxr-x. 1 777 root  1960 Sep 14 09:40 XX
-rwxrwxr-x. 1 777 root  1686 Sep 14 09:40 XX
-rwxrwxr-x. 1 777 root  4349 Sep 14 09:40 XX
-rwxrwxr-x. 1 777 root   965 Sep 14 09:40XX
drwxrwxr-x. 6 777 root  4096 Sep 14 09:40 XX
drwxrwxr-x. 2 777 root 16384 Sep 14 09:40 XX 

<img width="1148" alt="Screenshot 2021-09-15 at 09 30 49" src="https://user-images.githubusercontent.com/45966029/133390392-77f49725-c2d2-433a-abf2-472da4214a51.png">

Thanks a lot for your help.

**Comment by drakkan** [comment]
Hi,

SFTPGo runs, by default, using the dedicated `sftpgo` user, please try:

```
chown -R sftpgo:sftpgo /sftp_drakkan
```

**Reply by drakkan** [reply]
Please note that if you are using the official rpm package a `users_base_dir` if defined, so you can just leave the home dir empty and it will be automatically set to a writable location

**Reply by charlotattardjpj** [reply]
Thanks a lot mate, I chowned the directory structure with stftpgo user and it worked before it was set to root.
